applications/nrf9160dk_demo/src/ui/include/ui.h

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**@file
+ *
+ * @brief   User interface module.
+ *
+ * Module that handles user interaction through button, RGB LED and buzzer.
+ */
+
+#ifndef UI_H__
+#define UI_H__
+
+#include <zephyr.h>
+#include <dk_buttons_and_leds.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define UI_BUTTON_1			1
+#define UI_BUTTON_2			2
+#define UI_SWITCH_1			3
+#define UI_SWITCH_2			4
+
+#define UI_LED_1			1
+#define UI_LED_2			2
+#define UI_LED_3			3
+#define UI_LED_4			4
+
+#define UI_LED_ON(x)			(x)
+#define UI_LED_BLINK(x)			((x) << 8)
+#define UI_LED_GET_ON(x)		((x) & 0xFF)
+#define UI_LED_GET_BLINK(x)		(((x) >> 8) & 0xFF)
+
+#define UI_LED_ON_PERIOD_NORMAL		500
+#define UI_LED_OFF_PERIOD_NORMAL	500
+
+/**@brief UI LED state pattern definitions. */
+enum ui_led_pattern {
+	UI_LTE_DISCONNECTED	= UI_LED_ON(0),
+	UI_LTE_CONNECTING	= UI_LED_BLINK(DK_LED3_MSK),
```

Comment:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/2177 has some minor LED changes, it would be good to pull those in here.

Reply:
Missed this earlier today so just updated it.  Thanks.